### PR TITLE
update README to clarify hash functionality not needed

### LINKED ISSUE #5: Unclear instructions on whether custom Enumerable module should be usable on non-Array objects
Working through the curriculum, I felt that the instructions were very ambiguous about whether I should implement the Enumerable methods to be compatible with non-Array data such as Hashes. The README states that people should
> reimplement common methods used frequently through the enumerables module with **objects like arrays and hashes**

While implementing the methods to work with arrays is reasonable considering the course progress, making it compatible with hashes increases the difficulty significantly. Considering none of the test cases actually test with hashes, I wouldn't be surprised if the intention was for the methods to only work on Arrays.

If so, an _explicit comment stating only Array compatibility is necessary_ would greatly clear up any confusion regarding the task given

### PR DESCRIPTION
**Pull request by Forthoney**
Added a sentence clarifying the scope of the task to just Arrays in line 67 of README.md
Fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ again to see if your code is correct or not.
 
 ### Methods
 
-The following table has the methods you will need to implement along with their associated specs, as outlined in the project instructions.
+The following table has the methods you will need to implement along with their associated specs, as outlined in the project instructions. The methods in the Enumerable module do not need to work with non-Arrays, but you can challenge yourself and try to make them work for Hashes as well.
```
Comment (ChargrilledChook) [inline README.md]:
```suggestion
The following table has the methods you will need to implement along with their associated specs, as outlined in the project instructions. The methods in the Enumerable module only need to work with Arrays, but you can challenge yourself and try to make them work for Hashes as well.
```
Great idea, but I think it flows a little better without the slight double negative. What do you think?

**Review by ChargrilledChook — CHANGES_REQUESTED** [review]
Apologies for the delay getting back to you on this one @Forthoney. Great change, I only have one small suggestion. Let me know your thoughts then we can get this one merged :rocket:

**Comment by Forthoney** [comment]
@ChargrilledChook Sounds much clearer! Thanks for the feedback

**Review by ChargrilledChook — APPROVED** [review]
Looks great, thanks @Forthoney!

**Pull request merged**